src/store/reducers/packagesReducer.js

```diff
@@ -106,6 +107,7 @@ export const redux = createReducerScenario({
   deletePackage: { type: 'PACKAGES_DELETE', reducer: deletePackageReducer },
   updatePackages: { type: 'PACKAGES_UPDATE', reducer: updatePackagesReducer },
   registerError: { type: 'PACKAGES_REDUCER_ERROR', reducer: errorReducer },
+  resetPendingAndError: { type: 'PACKAGES_RESET_ERROR_PENDING', reducer: resetPendingAndErrorReducer },
```

Comment:
see comment above about REDUCER in type name

Reply:
done